tetris.js

```diff
@@ -74,7 +74,9 @@ module.exports = {
 			}
 			
 			for(let i = 0; i < arrCoordinates_X.length; i++){
-				if(field[coordinates_Y[i]][coordinates_X[i]] == true){
```

Comment:
This function has several problems:
1. It never returns false, only undedefined - please make it so it only returns true or false, nothing else
2. it's not clear what is the type of elem? Is is single element? Is is array of elements? Rename parameter and add commentary above the function that explains it's purpose